modules/commons/src/test/java/org/apache/synapse/commons/json/JsonValueTest.java

```diff
@@ -43,35 +46,60 @@
     private static final String failTestVal5 = "-abc";
     private static final String failTestVal6 = "\"Hello world!";
 
+    /**
+     * Tests JSON value enclosed in double quotes
+     */
     public void testCase1() {
         runTest(stringVal);
     }
 
+    /**
+     * Tests JSON value null
+     */
     public void testCase2() {
         runTest(nullVal);
     }
 
+    /**
+     * Tests JSON value true
+     */
     public void testCase3() {
         runTest(trueVal);
     }
 
+    /**
+     * Tests JSON value false
+     */
     public void testCase4() {
         runTest(falseVal);
     }
 
+    /**
+     * Tests JSON value number
+     */
     public void testCase5() {
         runTest(numberVal1);
     }
 
+    /**
+     * Tests JSON value number
+     */
     public void testCase6() {
         runTest(numberVal2);
     }
 
+    /**
+     * Tests JSON value number
```

Comment:
What's the difference between this and the method above? Shall we have the doc, in a way that the test is clear?